testsuite/ui/widgets/oas3.py

```diff
@@ -0,0 +1,117 @@
+"""Widgets used by OAS3"""
+from selenium.webdriver.common.by import By
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import ParametrizedView, Text, Table, Widget, TextInput, GenericLocatorWidget
+
+
+# pylint: disable=abstract-method
+
+
+class OAS3DropDown(Widget):
+    """DropDown element used by OAS3. Usually used for key selection or autocomplete function"""
+
+    EXPAND = ".//select"
+    OPTION = './/select/option[contains(text(),"{}")]'
+
+    def __init__(self, parent=None, locator='.//div[@class="examples-select"]', logger=None):
+        super().__init__(parent, locator=locator, logger=logger)
+        self.locator = locator
+
+    def select_item(self, value):
+        """Selects item by their respective values in the select"""
+        item = self.OPTION.format(value)
+        self.browser.selenium.find_element(By.XPATH, item).click()
+
+    def __locator__(self):
+        return self.locator
+
+    def __repr__(self):
+        return "{}{}".format(type(self).__name__, self.locator)
+
+
+class Endpoint(ParametrizedView):
```

Comment:
I think that this view should be located with other views and not with widgets.

Reply:
The View is Widget. `/ui/wdigets` contains elements used across different page objects across the UI test suite. `class Endpoint` is also designed for multiple page objects. 
It does not make sense to move it to the View package.

Reply:
I agree with Smolar, it makes more sense to have it as a generic endpoint widget as it is present on multiple pages